server/events/vcs/github_client.go

```diff
@@ -429,6 +429,18 @@ func (g *GithubClient) UpdateStatus(ctx context.Context, request types.UpdateSta
 	return err
 }
 
+// UpdateChecksStatus updates the status check
+func (g *GithubClient) UpdateChecksStatus(ctx context.Context, request types.UpdateStatusRequest) error {
```

Comment:
We should remove this when statuses are migrated to github checks.

Reply:
Yup, I have created a ticket for removing the ff flags in our backlog and referenced it in the TODO comments.